winml/lib/Api/LearningModel.cpp

```diff
@@ -22,24 +20,19 @@ WINML_CATCH_ALL
 LearningModel::LearningModel(
     const std::string& path,
     const winml::ILearningModelOperatorProvider operator_provider) try : operator_provider_(operator_provider) {
-  _winmlt::PerformanceTelemetryEvent kLoadModel_event(
-      WinMLRuntimePerf::kLoadModel);
```

Comment:
This event doesn't send up telemetry. It merely logs a runtime ETW event. This is because there is no telemetry keyword in the TraceLoggingWrite call. An example of telemetry keyword is: MICROSOFT_KEYWORD_MEASURES, MICROSOFT_KEYWORD_TELEMETRY, or MICROSOFT_KEYWORD_CRITICAL_DATA

Reply:
great catch !    this was how we were getting the start/stop etw tracing.  WINML_PROVIDER_KEYWORD_START_STOP

---
In reply to: [355714030](https://github.com/microsoft/onnxruntime/pull/2577#discussion_r355714030) [](ancestors = 355714030)